tests/system/action/user/test_create.py

```diff
@@ -290,68 +346,236 @@ def test_create_permission_superadmin(self) -> None:
+        )
+
+        response = self.request(
+            "user.create",
+            {
+                "username": "new username",
+                "title": "new title",
+                "first_name": "new first_name",
+                "last_name": "new last_name",
+                "is_active": True,
+                "is_physical_person": True,
+                "default_password": "new default_password",
+                "gender": "female",
+                "email": "info@openslides.com ",
+                "default_number": "new default_number",
+                "default_structure_level": "new default_structure_level",
+                "default_vote_weight": "1.234000",
+                "group_$_ids": {"1": [1], "4": [4]},
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "user/3",
+            {
+                "username": "new username",
+                "title": "new title",
+                "first_name": "new first_name",
+                "last_name": "new last_name",
+                "is_active": True,
+                "is_physical_person": True,
+                "default_password": "new default_password",
+                "gender": "female",
+                "email": "info@openslides.com ",
+                "default_number": "new default_number",
+                "default_structure_level": "new default_structure_level",
+                "default_vote_weight": "1.234000",
+                "group_$1_ids": [1],
+                "group_$4_ids": [4],
+            },
+        )
+
+    def test_create_permission_group_A_no_permission(self) -> None:
+        """ May not create group A fields on organsisation scope, although having both committee permissions"""
         self.permission_setup()
         self.create_meeting(base=4)
-        self.set_user_groups(self.user_id, [6])
         self.update_model(
             f"user/{self.user_id}",
             {
-                "committee_ids": [60, 63],
-                "organisation_management_level": OrganisationManagementLevel.CAN_MANAGE_USERS,
+                "committee_$60_management_level": CommitteeManagementLevel.CAN_MANAGE,
+                "committee_$63_management_level": CommitteeManagementLevel.CAN_MANAGE,
+                "committee_$_management_level": ["60", "63"],
             },
         )
 
         response = self.request(
             "user.create",
             {
-                "username": "username",
-                "group_$_ids": {1: [1], 4: [5]},
+                "username": "new username",
+                "committee_ids": [60],
+                "group_$_ids": {"4": [4]},
+            },
+        )
+        self.assert_status_code(response, 403)
+        self.assertIn(
+            "You are not allowed to perform action user.create. Missing permission: OrganisationManagementLevel can_manage_users in organisation 1",
+            response.json["message"],
+        )
+
```

Comment:
For group A I would like to have 2 small success tests (not many fields) for a create with the right CML and meeting perms.

Reply:
done